docs/source/contrib_guide_code.rst

```diff
@@ -0,0 +1,80 @@
+==================================
+Contributing to the Code
+==================================
+
+How can I help?
+---------------
+Individuals are welcome, and encouraged, to submit issues, enhancements, and
+new idea to the codebase. If you are a first-time contributor looking to get
+involved with Jupyter, you can use the following query in a Github search to
+find beginner-friendly issues to tackle across the Jupyter codebase.
+
+**is:issue is:open is:sprint-friendly user:jupyter**
+
+Once you've found an issue that you are eager to solve, you can use the guide
+below to get started. If you experience any issues while working on the issue,
```

Comment:
`issue` twice in 1 line. I would suggest people to comment on the issue that to flood mailing list and gitter channel. 
It also leave a trace for futur reviewer, people that want to tackle the issue later if it is not fixed. 

Assume gitter is `/dev/null` after 48h.